src/components/Tipseen/Tipseen.tsx

```diff
@@ -146,7 +146,7 @@ const Tipseen: VibeComponent<TipseenProps> & {
           <div className={cx(styles.tipseenContent)}>{content}</div>
         </div>
       ),
-      [content, onClose, overrideCloseAriaLabel, overrideHideCloseButton, title, titleClassName]
+      [closeButtonTheme, content, onClose, overrideCloseAriaLabel, overrideHideCloseButton, title, titleClassName]
```

Comment:
The fix is ok, but is this `useMemo` really needed?
I don't think there's so much rendering happening in this component so maybe it would just be simpler to render it regularly as node instead of a function

Reply:
@hadasfa it was written by you, so what do you think?

Reply:
I think it is ok to remove,  I don't remember why I added it.
Just worth checking it's not causing weird rerendering issues (but I think it will be ok)